deployments/bridges/rialto-millau/entrypoints/relay-messages-to-millau-generator-entrypoint.sh

```diff
@@ -58,4 +64,49 @@ do
 		transfer \
 		--amount 1000000000 \
 		--recipient $FERDIE_ADDR
+
+	# every other hour we're sending 3 large (size, weight, size+weight) messages
+	if [ $SECONDS -ge $LARGE_MESSAGES_TIME ]; then
+		LARGE_MESSAGES_TIME=$((SECONDS + 7200))
```

Comment:
I know it's not any moonmath, but having:

```bash
TWO_HOURS=7200
```

Would be nice imo

Reply:
The variable is an overkill, if we want to make it more explicit I'd rather go with `SECONDS + 2 * 60 * 60`, but it's fine as-is for me.